browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2975,6 +2975,12 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			break;
 
 		case 'enable':
+			// E.g. GtkEntry is represented as an <input> inside a <div>, the disabled
+			// attribute is on the <input>, not the <div>.
+			var innerInput = control.querySelector('input');
```

Comment:
ah, we have better solution for that -> the widget itself doesn't update inner input on root change.
Let me push update here so you can test